cassovary-core/src/main/scala/com/twitter/cassovary/graph/SharedArrayBasedDirectedGraph.scala

```diff
@@ -66,10 +66,10 @@ object SharedArrayBasedDirectedGraph {
      */
     log.info("read out num of edges and max id from files in parallel")
     val futures = statsReceiver.time("graph_dump_read_out_num_of_edge_and_max_id_parallel") {
-      def readNumOfEdgesAndMaxId(iteratorFunc: () => Iterator[NodeIdEdgesMaxId]) =
+      def readNumOfEdgesAndMaxId(iteratorFunc: Iterable[NodeIdEdgesMaxId]) =
```

Comment:
change iteratorFunc to some other name: edgesIterable or something else you like